### PR TITLE
SmthJedi audio updates

### PR DESCRIPTION
**Pull request by NoSloppy**
Added boot.wav.
Added endlock made from lock + SmthGrey clash04.
Normalized hum to -5db and high passed @100hz, 48db slope.
Normalized lock to 0db and high passed @100hz, 48db slope.

### CODE REVIEW

**Comment by profezzorn** [comment]
Where did boot.wav come from?
Where the files comes from should be documented in the readme.txt file IMHO.
I don't think we should be doing any highpass filtering, that can be done on the saber.

**Comment by NoSloppy** [comment]
> Where did boot.wav come from? Where the files comes from should be documented in the readme.txt file IMHO. 

This is covered with the line in the ReadMe.txt "Any additional sounds sourced from creative-commons licensed freesound.org."

>I don't think we should be doing any highpass filtering, that can be done on the saber.

Replaced high pass versions with full range.

**Comment by profezzorn** [comment]
> Any additional sounds sourced from creative-commons licensed freesound.org."

I think the right way to do it is to add a specific entry for boot.wav to the text file *in this PR*.
That helps future font readers to know where everything came from, and who added it, and it also helps me as a reviewer.

File: SmthJedi/license.txt [hunk 1]
```diff
@@ -1 +1,11 @@
-These files are free to use, modify, and redistribute without attribution or any other restrictions.
\ No newline at end of file
+These files are free to use, modify, and redistribute without attribution or any other restrictions.
+
+Modifications by Brian Conner 2022:
+Detailed changes can be found in this font's ReadMe.txt.
+Any additional sounds sourced from creative-commons licensed freesound.org.
```
Comment (profezzorn) [inline SmthJedi/license.txt]: Please don't say "any sounds added ...."
Please specify which sounds were added and fall under this license.

Comment (NoSloppy) [reply SmthJedi/license.txt]: Done.

**Pull request merged**